br/pkg/lightning/backend/external/writer.go

```diff
@@ -320,6 +320,7 @@ type Writer struct {
 // WriteRow implements ingest.Writer.
 func (w *Writer) WriteRow(ctx context.Context, idxKey, idxVal []byte, handle tidbkv.Handle) error {
 	keyAdapter := w.keyAdapter
+	w.batchSize += uint64(len(idxKey) + len(idxVal) + 2*lengthBytes)
```

Comment:
What is the different between here and line 349?

```
323: w.batchSize += uint64(len(idxKey) + len(idxVal) + 2*lengthBytes)

349: w.batchSize += uint64(length)
```

Reply:
I just merged master, maybe there are some mistakes 😂